core/sqlite3_ondisk.rs

```diff
@@ -35,8 +35,9 @@ use std::rc::Rc;
 
 /// The size of the database header in bytes.
 pub const DATABASE_HEADER_SIZE: usize = 100;
```

Comment:
Let's also add a `MIN_PAGE_CACHE_SIZE` constant of `10` to follow SQLite.

Reply:
done!